CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 
+- feat: Add read-only scope property to Hub #975
```

Comment:
Is this just a refactor though? use scope property in favor of getScope?
What about adding a migration guide note for users that the old API is marked obsolete?

```suggestion
- ref: Add read-only scope property to Hub #975
```